ctxscope/testdata/src/a/a.go

```diff
@@ -25,6 +25,8 @@ func (datastoreInterface) Delete(c context.Context, key interface{}) error {
 
 var datastore = datastoreInterface{}
 
+var ctx, _ = context.WithCancel(context.TODO())
```

Comment:
I think there is no need to define as `ctx`, instead with `_`

Reply:
Good point, updated!